MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h

```diff
@@ -0,0 +1,52 @@
+/**
+ Description: Utility function to calculate the Magnetic Field on the GPU
+*/
+
+#ifndef MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+#define MagneticField_ParametrizedEngine_interface_alpaka_ParabolicParametrizedMagneticField_h
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  namespace MagneticFieldParabolicPortable {
+
+    struct Parameters {
+      float c1 = 3.8114;
+      float b0 = -3.94991e-06;
+      float b1 = 7.53701e-06;
+      float a = 2.43878e-11;
+      float max_radius2 = 13225.f;  // tracker radius
+      float max_z = 280.f;          // tracker z
```

Comment:
Is there a reason why these parameters should be regular data members, instead of `static constexpr` ?

```suggestion
      static constexpr float c1 = 3.8114;
      static constexpr float b0 = -3.94991e-06;
      static constexpr float b1 = 7.53701e-06;
      static constexpr float a = 2.43878e-11;
      static constexpr float max_radius2 = 13225.f;  // tracker radius
      static constexpr float max_z = 280.f;          // tracker z
```

Then they could be used directly as `Parameters::a` without instantiating a `Parameters` object.

Reply:
Changed: https://github.com/cms-sw/cmssw/pull/44360/commits/a088e4686ce71a97ce88a7c87293e4d05854fda8